docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -10,6 +10,10 @@ aliases:
   - ../../../enterprise/saml/troubleshoot-saml/
 description: Learn how to configure SAML authentication in Grafana's configuration file.
 menuTitle: Configure SAML using the configuration file
+labels:
+  products:
+    - cloud
+    - enterprise
```

Comment:
Good catch! Could you please add this to the [SAML UI docs](https://github.com/grafana/grafana/blob/main/docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md) as well 🙏